crates/shared/src/bad_token/trace_call.rs

```diff
@@ -580,18 +564,40 @@ mod tests {
         //   Not sure why deny listed.
 
         let settlement = contracts::GPv2Settlement::deployed(&web3).await.unwrap();
-        let uniswap = Arc::new(UniswapLikePairProviderFinder {
-            inner: uniswap_v2::get_liquidity_source(&web3).await.unwrap().0,
-            base_tokens: base_tokens.to_vec(),
-        });
-        let sushiswap = Arc::new(UniswapLikePairProviderFinder {
-            inner: sushiswap::get_liquidity_source(&web3).await.unwrap().0,
-            base_tokens: base_tokens.to_vec(),
+        let finder = Arc::new(TokenOwnerFinder {
+            web3: web3.clone(),
+            proposers: vec![
+                Arc::new(UniswapLikePairProviderFinder {
+                    inner: uniswap_v2::get_liquidity_source(&web3).await.unwrap().0,
+                    base_tokens: base_tokens.to_vec(),
+                }),
+                Arc::new(UniswapLikePairProviderFinder {
+                    inner: sushiswap::get_liquidity_source(&web3).await.unwrap().0,
+                    base_tokens: base_tokens.to_vec(),
+                }),
+                Arc::new(BalancerVaultFinder(
+                    BalancerV2Vault::deployed(&web3).await.unwrap(),
+                )),
+                Arc::new(
+                    UniswapV3Finder::new(
+                        IUniswapV3Factory::deployed(&web3).await.unwrap(),
+                        base_tokens.to_vec(),
+                        web3.eth().block_number().await.unwrap().as_u64(),
+                        FeeValues::Dynamic,
+                    )
+                    .await
+                    .unwrap(),
+                ),
+                Arc::new(
+                    BlockscoutTokenOwnerFinder::try_with_network(reqwest::Client::new(), 1)
+                        .unwrap(),
```

Comment:
An error from `BlockscoutTokenOwnerFinder::try_with_network()` was silently ignored before. Now it would panic.

Reply:
Note this is an integration test - we used to not load up the `BlockscoutTokenOwnerFinder` in the integration test before, but I added some known "bad" tokens that needed it for finding token holders with balance.